frontend/src/scenes/session-recordings/player/list/listLogic.ts

```diff
@@ -91,8 +94,9 @@ export const listLogic = kea<listLogicType>([
     })),
     selectors(({ props }) => ({
         data: [
-            () => [TAB_TO_LOGIC[props.tab]?.selectors.data],
-            (data: Record<string, any>[]): Record<string, any>[] => data ?? [],
+            (selectors) => [selectors.eventListData, selectors.consoleListData],
+            (eventListData, consoleListData): Record<string, any>[] =>
+                props.tab === SessionRecordingTab.CONSOLE ? consoleListData : eventListData,
```

Comment:
This is the notable line - everything else is just to rename `data` -> `consoleListData` and `data` -> `eventListData` because otherwise, the names would conflict